app/routes/docs.samples.mdx

```diff
@@ -0,0 +1,176 @@
+---
+handle:
+  breadcrumb: Code Samples
+---
+
+import { Highlight } from '~/components/Highlight'
+
+# Code Samples
+
+We have collected a short list of functions and examples that may be useful
+when working with GCN Notices in your code. These include conversions from
+text/VOEvent alert formats to json and xml {'('}for VOEvent alerts{')'} and
+examples for how to save alerts in these formats.
+
+For more examples, or to contribute your own ideas, see our [Utility Sample](https://github.com/nasa-gcn/utility-samples) and [gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python) repositories.
+
+## Parsing
+
+Within your consumer loop, the following functions can be used to convert the
+content of <code>message.value()</code> into other data types.
+
+<code>xmltodict</code> is not a standard python package, it must be installed with
+pip:
+
+<Highlight language="sh" code="pip install xmltodict" />
+
+```python
+
+import email
+import xml.etree.ElementTree as ET
+import xmltodict
+
+def parse_text_alert_to_dict(message_value):
+    return dict(email.message_from_bytes(message_value))
+
+def parse_voevent_alert_to_xml_root(message_value):
+    return ET.fromstring(message_value)
+
+def parse_voevent_alert_to_dict(message_value):
+    return xmltodict.parse(message_value)
+
+```
+
+## Saving
+
+Once parsed, the following can be used to save the data to a local file.
```

Comment:
You don't need to parse the alert to save it to a file. In fact, generally it's easier not to.